bundle-ocp/manifests/das-operator.clusterserviceversion.yaml

```diff
@@ -6,103 +6,168 @@ metadata:
-    mediatype: image/png
+  - base64data: ""
```

Comment:
You may want to file a story here or try to keep this change.

Otherwise, you will end up with no icon for your operator.